infra/README.md

```diff
@@ -9,35 +9,57 @@ Run these steps the first time.
    see [AWS Configuration basics](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config)
    and [Where are the configuration settings stored](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
    for more info)
-3. To manage existing instances, install [sops](   ) to decrypt encrypted variable files in the repository.
+3. To manage existing instances, install [sops](https://github.com/getsops/sops) to decrypt encrypted variable files in the repository.
 
 ### Multi-factor authentication (MFA)
 
-If you get a 403 error when deploying or destroying resources with terraform despite having configured a valid AWS
+If you get a 403 error when running terraform despite having configured a valid AWS
 access key, you may need to set up MFA. Install both AWS CLI and jq, and make sure you have `aws` and `jq` in path. Execute the `[get-mfa-vars.sh](https://gist.github.com/mvaaltola/0abced5790401f2454444fb2ffd4acc0)` script with your AWS arn and your MFA access code,
```

Comment:
I know it is not addition in this PR, but considering the instructions overall, it might clearer if explicitly stated that it is the aws arn related to the mfa device?